plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -87,4 +125,24 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
   }
 }
 
-module.exports = NodesTableContainer;
+const getGraphQl = () => {
```

Comment:
Why do we have this as a function? RxJS observables are lazily invoked, so it could also be a constant, right?

With the current implementation, wouldn't componentFromStream call the inner callback every time it gets new props, which means it would unsubscribe from the old stream and subscribe to the new one returned by `getGraphql()` again?

Reply:
👍